Stratis.Bitcoin/Features/BlockStore/LoopSteps/ProcessPendingStorageStep.cs

```diff
@@ -46,24 +44,13 @@ internal ProcessPendingStorageStep(BlockStoreLoop blockStoreLoop, ILoggerFactory
         {
             this.logger.LogTrace("({0}:'{1}/{2}',{3}:{4})", nameof(nextChainedBlock), nextChainedBlock.HashBlock, nextChainedBlock.Height, nameof(disposeMode), disposeMode);
 
-            if (!this.BlockStoreLoop.PendingStorage.TryRemove(nextChainedBlock.HashBlock, out this.pendingBlockPairToStore))
-            {
-                this.logger.LogTrace("(-):{0}", StepResult.Next);
+            BlockPair pendingBlockPairToStore;
+            if (!this.BlockStoreLoop.PendingStorage.TryRemove(nextChainedBlock.HashBlock, out pendingBlockPairToStore))
```

Comment:
now that the IsInitialBlockDownload  is removed what happens when we are at the tip and a new block is arrived we will  not push it immediately right?